src/index.html

```diff
@@ -19,9 +19,402 @@
+          </div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+            </div>
+            <div class="card__review-rating">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-sum">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__buttom"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            src="../src/images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__product">
+            <p class="card__product-code">Product code: 195434</p>
+          </div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+            </div>
+            <div class="card__review-rating">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-sum">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__buttom"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div class="card cards__card">
+          <img
+            src="../src/images/imac.jpeg"
+            alt="imac"
+            class="card__img"
+          />
+          <h2 class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <div class="card__product">
+            <p class="card__product-code">Product code: 195434</p>
+          </div>
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+            </div>
+            <div class="card__review-rating">Reviews: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__price-label">Price:</div>
+            <div class="card__price-sum">$2,199</div>
+          </div>
+          <a
+            href="#"
+            class="card__buttom"
```

Comment:
There seems to be a typo in the class name 'card__buttom'. It should be 'card__button'.